src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/filter/AbstractUserFilter.java

```diff
@@ -39,6 +40,9 @@
  * @author Thomas Forrer
  */
 public abstract class AbstractUserFilter extends AbstractChangesFilter {
+    private static final String POPUP_TEXT = KeymapUtil.getShortcutsText(CommonShortcuts.CTRL_ENTER.getShortcuts()) +
```

Comment:
This line is quite long. Suggestion:
``java
    private static final String POPUP_TEXT = String.format("%s to search",
        KeymapUtil.getShortcutsText(CommonShortcuts.CTRL_ENTER.getShortcuts()));
``